packages/frontend/src/components/Explorer/SpaceBrowser/AddResourceToSpaceMenu.tsx

```diff
@@ -13,29 +13,22 @@ const AddResourceToSpaceMenu: React.FC<AddResourceToSpaceMenuProps> = ({
     resourceType,
     onAdd,
     onCreate,
-    children,
 }) => {
     return (
-        <Popover2
-            captureDismiss
-            position={PopoverPosition.BOTTOM_RIGHT}
-            content={
-                <Menu>
-                    <MenuItem2
-                        icon="plus"
-                        text={`Add existing ${resourceType}`}
-                        onClick={onAdd}
-                    />
-                    <MenuItem2
-                        icon="clean"
-                        text={`Create new ${resourceType}`}
-                        onClick={onCreate}
-                    />
-                </Menu>
-            }
-        >
-            {children}
-        </Popover2>
+        <Menu>
+            <MenuItem2
+                icon="plus"
+                text={`Add existing ${resourceType}`}
+                onClick={onAdd}
+                style={{ margin: '-5px', marginBottom: '0px' }}
```

Comment:
somebody told me we should never use inline styles... we should wrap this component and make a new component on styles.ts file